formcentral/Dockerfile

```diff
@@ -1,20 +1,16 @@
-FROM golang:alpine AS build
+FROM golang:alpine
+
 RUN apk add --no-cache \
-        gcc \
-        libc-dev
+    bash \
+    gcc \
+    libc-dev
+
 RUN mkdir /app
 WORKDIR /app
 
-ADD go.mod .
-ADD go.sum .
+COPY go.mod .
+COPY go.sum .
 RUN go mod download
-ADD . /app/
-RUN go build -o main .
-
 
-FROM alpine
-WORKDIR /app
-COPY --from=build /app/main /app/
-
-EXPOSE 1323
```

Comment:
@nandanrao, what do you think about letting the container operator/orchestrator set the port to the container? That way we can set it dynamically.

Reply:
Expose is really just there for documentation purposes, doesn't do anything given that we run it always in orchestration.